packages/main/src/ColorPaletteItem.ts

```diff
@@ -52,6 +52,19 @@ class ColorPaletteItem extends UI5Element implements ITabbable {
 	@property({ validator: CSSColor })
 	value!: string;
 
+	/**
+	 * Defines if the component is selected.
+	 * <br><br>
+	 * <b>Note:</b> Only one item must be selected per <code>ui5-color-palette</code>.
```

Comment:
If more than one item is defined as selected, the last one would be considered as the selected one.